dashboards/k8s-views-global.json

```diff
@@ -745,7 +745,7 @@
             "type": "prometheus",
             "uid": "${datasource}"
           },
-          "expr": "count(kube_namespace_created{cluster=\"$cluster\"})",
+          "expr": "sum(kube_namespace_status_phase{cluster=\"$cluster\",phase=\"Active\"})",
```

Comment:
Hi @rodrigoccaldas,

According to the kube-state-metrics [documentation](https://github.com/kubernetes/kube-state-metrics/blob/7104d579e93d672754c018a924d6c3f7ec23874e/docs/metrics/cluster/namespace-metrics.md), the `kube_namespace_created` metric is still in `STABLE` status.

Could you please provide the source of your information?